cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
can we lose this file?

Reply:
Gone! Also added it to gitignore since it's being added by default when you run cypress in headed mode